apps/fz_http/lib/fz_http_web/templates/layout/admin.html.heex

```diff
@@ -113,8 +104,8 @@
               </div>
               <div class="level-item">
                 <div class="logo">
-                  <a href="https://www.firezone.dev/?utm_source=product">
-                    <img src="/images/logo.svg" alt="firezone.dev" />
+                  <a href="https://firezone.dev?utm_source=product">
```

Comment:
(Nit) We’ve been trying to use the www subdomain to help distinguish website URLs from others for things like search and replace and referrer tracking. Not a biggie but may want to be consistent